### PR TITLE
Issue #384: Clarify Opus output sample rate should be 48 kHz

### PR DESCRIPTION
**Pull request by jwcullen**
***
<a href="https://pr-preview.s3.amazonaws.com/jwcullen/iamf/pull/385.html" title="Last updated on Jun 6, 2023, 1:00 AM UTC (11dfbbe)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/385/ba0e169...jwcullen:11dfbbe.html" title="Last updated on Jun 6, 2023, 1:00 AM UTC (11dfbbe)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1551,7 +1551,7 @@ For this version of the specification, the value of reinitialize_decoder shall b
 The difference, [=relative_offset=](P) - [=relative_offset=](A), shall be the offset to position the first audio sample where the first parameter value of the first parameter block that comes after this Sync OBU is applied to.
 - Where, [=relative_offset=](P) is [=relative_offset=] of [=obu_id=] for the parameter and [=relative_offset=](A) is [=relative_offset=] of [=obu_id=] for the substream which the parameter is applied to. 
 
-The offset shall be indicated in the number of ticks at the sample rate specified in the corresponding substream or Codec Config OBU. For IAMF-OPUS, the sample rate is 48000Hz.
+The offset shall be indicated in the number of ticks at the sample rate specified in the corresponding substream, Codec Config OBU, or [[#codec-specific]] section.
```
Comment (tdaede) [inline index.bs:1554]: Perhaps just say codec specific section, and then for each of our codecs, add a definition. For PCM, tick rate would be sample rate. Maybe even make up a variable name for this value.

**Review by tdaede — CHANGES_REQUESTED** [review]

File: index.bs [hunk 2]
```diff
@@ -1580,6 +1580,7 @@ For legacy codecs, Decoder_Config() shall be exactly the same information as the
 
 Substream format is [=opus packet=] of [[!RFC6716]] which contains only one single frame of mono or stereo channels and which has non-delimiting frame structure.
 
+An IA decoder must use a playback sample rate of 48 kHz as defined in [[!RFC7845]].
```
Comment (tdaede) [inline index.bs:1583]: "An IA decoder assumes a sample rate of 48kHz for all timing information."

Alternately, if we define a term such as tick rate:

"Opus substreams use a tick rate of 48kHz".

**Review by tdaede — CHANGES_REQUESTED** [review]

**Comment by tdaede** [comment]
If you want I can push changes on this PR.

**Comment by jwcullen** [comment]
@tdaede Yes that is fine. I have been focused on the test vectors, so I didn't get back to this yet.

**Comment by jwcullen** [comment]
Superseded by #439 and should be rejected once it is merged in.

**Pull request closed without merging**